README.md

```diff
@@ -50,6 +50,7 @@ Good luck! May you enjoy a peaceful day of good health and a night of happy drea
 | [1523586494154435](https://ordinals.com/sat/1523586494154435) | JESUS - SERMON ON THE MOUNT | TEXT | RELIGION |
 | [1225514403256383](https://ordinals.com/sat/1225514403256383) | AESOP - FABLES | TEXT | ART |
 | [1699518292891700](https://ordinals.com/sat/1699518292891700) | AESOP - PORTRAIT | IMAGE | ART |
+| [1015240021526433](https://ordinals.com/sat/1015240021526433) | A. A. MILNE - WINNIE THE POOH | TEXT | ART |
```

Comment:
maybe this isn't the right order in the list.  It was initially here when I was going to spell out the name "Alan Alexander Milne" and trying to keep alphabetical

Reply:
Fine, thank you. It can be sorted under MILNE